### PR TITLE
Adding trillium support for maxdiffusion tutorial

### PR DESCRIPTION
**Pull request by saltysoup**
This is to add trillium support for the [Serve Stable Diffusion XL (SDXL) using TPUs on GKE with MaxDiffusion](https://cloud.google.com/kubernetes-engine/docs/tutorials/serve-sdxl-tpu) by adding latest JAX + libtpu install binaries, and kubernetes deployment example to launch a trillium serving pod

### CODE REVIEW

**Review by NimJay — APPROVED** [review]
Looks good. I trust that you've done the appropriate testing.

Approving. Tag me if you'd like me to merge this.

File: ai-ml/maxdiffusion-tpu/build/server/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 uvicorn[standard]==0.30.6
 fastapi[all]==0.114.0
 pillow
-jax==0.4.31
-jaxlib==0.4.31
+#jax==0.4.31 # installed in Dockerfile
+#jaxlib==0.4.31 # installed in Dockerfile
```
Comment (NimJay) [inline ai-ml/maxdiffusion-tpu/build/server/requirements.txt:5]: **Question:** Is there a reason we now install these via the Dockerfile?

Comment (saltysoup) [reply ai-ml/maxdiffusion-tpu/build/server/requirements.txt]: @NimJay no, I've moved it into the requirements file with latest commit

Comment (saltysoup) [reply ai-ml/maxdiffusion-tpu/build/server/requirements.txt]: @NimJay please merge with main. Thank you

**Pull request merged**